--- glibc-2.6.1/configure.old	2013-02-12 05:10:28.614496246 +0100
+++ glibc-2.6.1/configure	2013-02-12 05:12:44.171568520 +0100
@@ -4463,7 +4463,7 @@
   # Found it, now check the version.
   echo "$as_me:$LINENO: checking version of $SED" >&5
 echo $ECHO_N "checking version of $SED... $ECHO_C" >&6
-  ac_prog_version=`$SED --version 2>&1 | sed -n 's/^.*GNU sed version \([0-9]*\.[0-9.]*\).*$/\1/p'`
+  ac_prog_version=`$SED --version 2>&1 | sed -n 's/^.*(GNU sed) \([0-9]*\.[0-9.]*\).*$/\1/p'`
   case $ac_prog_version in
     '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
     3.0[2-9]*|3.[1-9]*|[4-9]*)
