diff --git a/usr/include/stddef.h b/usr/include/stddef.h
index fde5ad9..ada14d3 100644
--- a/usr/include/stddef.h
+++ b/usr/include/stddef.h
@@ -61,4 +61,18 @@ typedef	__wchar_t	wchar_t;
 
 #define	offsetof(type, member)	__offsetof(type, member)
 
+#if (defined (__STDC_VERSION__) && __STDC_VERSION__ >= 201112L) \
+  || (defined(__cplusplus) && __cplusplus >= 201103L)
+#ifndef _GCC_MAX_ALIGN_T
+#define _GCC_MAX_ALIGN_T
+/* Type whose alignment is supported in every context and is at least
+   as great as that of any standard type not using alignment
+   specifiers.  */
+typedef struct {
+  long long __max_align_ll __attribute__((__aligned__(__alignof__(long long))));
+  long double __max_align_ld __attribute__((__aligned__(__alignof__(long double))));
+} max_align_t;
+#endif
+#endif /* C11 or C++11.  */
+
 #endif /* _STDDEF_H_ */
