--- freetype-2.4.11/modules.cfg.old	2013-05-15 08:02:30.804077747 +0200
+++ freetype-2.4.11/modules.cfg	2013-05-15 08:03:04.054080039 +0200
@@ -37,35 +37,35 @@
 # PostScript Type 1 font driver.
 #
 # This driver needs the `psaux', `pshinter', and `psnames' modules.
-FONT_MODULES += type1
+# FONT_MODULES += type1
 
 # CFF/OpenType font driver.
 #
 # This driver needs the `sfnt', `pshinter', and `psnames' modules.
-FONT_MODULES += cff
+# FONT_MODULES += cff
 
 # Type 1 CID-keyed font driver.
 #
 # This driver needs the `psaux', `pshinter', and `psnames' modules.
-FONT_MODULES += cid
+# FONT_MODULES += cid
 
 # PFR/TrueDoc font driver.  See optional extension ftpfr.c below also.
-FONT_MODULES += pfr
+# FONT_MODULES += pfr
 
 # PostScript Type 42 font driver.
 #
 # This driver needs the `truetype' and `psaux' modules.
-FONT_MODULES += type42
+# FONT_MODULES += type42
 
 # Windows FONT/FNT font driver.  See optional extension ftwinfnt.c below
 # also.
-FONT_MODULES += winfonts
+# FONT_MODULES += winfonts
 
 # PCF font driver.
-FONT_MODULES += pcf
+# FONT_MODULES += pcf
 
 # BDF font driver.  See optional extension ftbdf.c below also.
-FONT_MODULES += bdf
+# FONT_MODULES += bdf
 
 # SFNT files support.  If used without `truetype' or `cff', it supports
 # bitmap-only fonts within an SFNT wrapper.
@@ -94,7 +94,7 @@
 ####
 
 # Monochrome rasterizer.
-RASTER_MODULES += raster
+# RASTER_MODULES += raster
 
 # Anti-aliasing rasterizer.
 RASTER_MODULES += smooth
@@ -110,37 +110,37 @@
 AUX_MODULES += cache
 
 # TrueType GX/AAT table validation.  Needs ftgxval.c below.
-AUX_MODULES += gxvalid
+# AUX_MODULES += gxvalid
 
 # Support for streams compressed with gzip (files with suffix .gz).
 #
 # See include/freetype/ftgzip.h for the API.
-AUX_MODULES += gzip
+# AUX_MODULES += gzip
 
 # Support for streams compressed with LZW (files with suffix .Z).
 #
 # See include/freetype/ftlzw.h for the API.
-AUX_MODULES += lzw
+# AUX_MODULES += lzw
 
 # Support for streams compressed with bzip2 (files with suffix .bz2).
 #
 # See include/freetype/ftbzip2.h for the API.
-AUX_MODULES += bzip2
+# AUX_MODULES += bzip2
 
 # OpenType table validation.  Needs ftotval.c below.
 #
-AUX_MODULES += otvalid
+# AUX_MODULES += otvalid
 
 # Auxiliary PostScript driver component to share common code.
 #
 # This module depends on `psnames'.
-AUX_MODULES += psaux
+# AUX_MODULES += psaux
 
 # Support for PostScript glyph names.
 #
 # This module can be controlled in ftconfig.h
 # (FT_CONFIG_OPTION_POSTSCRIPT_NAMES).
-AUX_MODULES += psnames
+# AUX_MODULES += psnames
 
 
 ####
@@ -155,7 +155,7 @@
 # Access BDF-specific strings.  Needs BDF font driver.
 #
 # See include/freetype/ftbdf.h for the API.
-BASE_EXTENSIONS += ftbdf.c
+# BASE_EXTENSIONS += ftbdf.c
 
 # Utility functions for converting 1bpp, 2bpp, 4bpp, and 8bpp bitmaps into
 # 8bpp format, and for emboldening of bitmap glyphs.
@@ -166,17 +166,17 @@
 # Access CID font information.
 #
 # See include/freetype/ftcid.h for the API.
-BASE_EXTENSIONS += ftcid.c
+# BASE_EXTENSIONS += ftcid.c
 
 # Access FSType information.  Needs fttype1.c.
 #
 # See include/freetype/freetype.h for the API.
-BASE_EXTENSIONS += ftfstype.c
+# BASE_EXTENSIONS += ftfstype.c
 
 # Support for GASP table queries.
 #
 # See include/freetype/ftgasp.h for the API.
-BASE_EXTENSIONS += ftgasp.c
+# BASE_EXTENSIONS += ftgasp.c
 
 # Convenience functions to handle glyphs.  Needs ftbitmap.c.
 #
@@ -186,12 +186,12 @@
 # Interface for gxvalid module.
 #
 # See include/freetype/ftgxval.h for the API.
-BASE_EXTENSIONS += ftgxval.c
+# BASE_EXTENSIONS += ftgxval.c
 
 # Support for LCD color filtering of subpixel bitmaps.
 #
 # See include/freetype/ftlcdfil.h for the API.
-BASE_EXTENSIONS += ftlcdfil.c
+# BASE_EXTENSIONS += ftlcdfil.c
 
 # Multiple Master font interface.
 #
@@ -201,7 +201,7 @@
 # Interface for otvalid module.
 #
 # See include/freetype/ftotval.h for the API.
-BASE_EXTENSIONS += ftotval.c
+# BASE_EXTENSIONS += ftotval.c
 
 # Support for FT_Face_CheckTrueTypePatents.
 #
@@ -211,34 +211,34 @@
 # Interface for accessing PFR-specific data.  Needs PFR font driver.
 #
 # See include/freetype/ftpfr.h for the API.
-BASE_EXTENSIONS += ftpfr.c
+# BASE_EXTENSIONS += ftpfr.c
 
 # Path stroker.  Needs ftglyph.c.
 #
 # See include/freetype/ftstroke.h for the API.
-BASE_EXTENSIONS += ftstroke.c
+# BASE_EXTENSIONS += ftstroke.c
 
 # Support for synthetic embolding and slanting of fonts.  Needs ftbitmap.c.
 #
 # See include/freetype/ftsynth.h for the API.
-BASE_EXTENSIONS += ftsynth.c
+# BASE_EXTENSIONS += ftsynth.c
 
 # Interface to access data specific to PostScript Type 1 and Type 2 (CFF)
 # fonts.
 #
 # See include/freetype/t1tables.h for the API.
-BASE_EXTENSIONS += fttype1.c
+# BASE_EXTENSIONS += fttype1.c
 
 # Interface for accessing data specific to Windows FNT files.  Needs winfnt
 # driver.
 #
 # See include/freetype/ftwinfnt.h for the API.
-BASE_EXTENSIONS += ftwinfnt.c
+# BASE_EXTENSIONS += ftwinfnt.c
 
 # Support functions for X11.
 #
 # See include/freetype/ftxf86.h for the API.
-BASE_EXTENSIONS += ftxf86.c
+# BASE_EXTENSIONS += ftxf86.c
 
 ####
 #### The components `ftsystem.c' (for memory allocation and stream I/O
